tuning/sft_trainer.py

```diff
@@ -228,6 +228,7 @@ def train(
         special_tokens_dict=special_tokens_dict,
         tokenizer=tokenizer,
         model=model,
+        multiple=model_args.embedding_size_multiple,
```

Comment:
maybe `multiple_of` instead of `multiple` is more readible?

Reply:
I agree, modified!